mico-core/src/test/java/io/github/ust/mico/core/ApplicationControllerTest.java

```diff
@@ -93,11 +93,24 @@ public void createApplication() throws Exception {
         given(applicationRepository.save(any(Application.class))).willReturn(application);
 
         final ResultActions result = mvc.perform(post(BASE_PATH)
-            .content(mapper.writeValueAsBytes(application))
-            .contentType(MediaTypes.HAL_JSON_UTF8_VALUE))
-            .andDo(print());
+                .content(mapper.writeValueAsBytes(application))
+                .contentType(MediaTypes.HAL_JSON_UTF8_VALUE))
+                .andDo(print());
 
         result.andExpect(status().isCreated());
     }
 
+    @Test
+    public void deleteApplication() throws Exception {
+        Application application = new Application(SHORT_NAME, VERSION, DESCRIPTION);
+
+        given(applicationRepository.findByShortNameAndVersion(SHORT_NAME, VERSION)).willReturn(Optional.of(application));
+
+        ResultActions resultDelete = mvc.perform(delete(BASE_PATH + SHORT_NAME + "/" + VERSION)
```

Comment:
We might think of introducing a new variable for `BASE_PATH + SHORT_NAME + "/" + VERSION`.